modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/fragment/renderer/ViewVersionHistoryJSPFragmentRenderer.java

```diff
@@ -31,16 +34,28 @@ public String getCollectionKey() {
 	protected ViewVersionHistoryDisplayContext getDisplayContext(
 		HttpServletRequest httpServletRequest) {
 
-		return new ViewVersionHistoryDisplayContext(
-			httpServletRequest, _objectDefinitionLocalService,
-			_objectEntryService);
+		try {
+			return new ViewVersionHistoryDisplayContext(
+				httpServletRequest, _objectDefinitionLocalService,
+				_objectEntryService);
+		}
+		catch (PortalException portalException) {
+			if (_log.isDebugEnabled()) {
+				_log.debug(portalException);
+			}
+		}
```

Comment:
If this is a NoSuch* exception, can you let it fail instead of swallowing the error?